fms/models/roberta.py

```diff
@@ -275,7 +275,7 @@ def reset_parameters(self):
     emb_dim=192, nheads=4, nlayers=5, max_pos=1024, src_vocab_size=256
 )
 
-_base_config = RoBERTaConfig()
+_base_config = RoBERTaConfig(tie_heads=True, norm_eps=1e-5, p_dropout=0.1)
```

Comment:
are our defaults wrong?

Reply:
I believe they were for this config (at least compared to roberta-base), in my roberta tests I needed to override these.